getwork.go

```diff
@@ -3,182 +3,14 @@
 package main
 
 import (
-	"bytes"
-	"crypto/tls"
-	"crypto/x509"
-	"encoding/binary"
-	"encoding/hex"
 	"encoding/json"
 	"fmt"
-	"io"
-	"math/big"
-	"net"
-	"net/http"
-	"os"
 	"strconv"
-	"time"
 
-	"github.com/decred/go-socks/socks"
 	"github.com/decred/gominer/stratum"
-	"github.com/decred/gominer/util"
 	"github.com/decred/gominer/work"
 )
 
-// newHTTPClient returns a new HTTP client that is configured according to the
-// proxy and TLS settings in the associated connection configuration.
-func newHTTPClient(cfg *config) (*http.Client, error) {
```

Comment:
I believe you're still going to need this to be able to talk to the stratum pools.